dashboard/app/controllers/activities_controller.rb

```diff
@@ -57,8 +57,9 @@ def milestone
     if params[:program] && sharing_allowed
       share_failure = nil
       if @level.game.sharing_filtered?
+        project_type = 'playlab'
```

Comment:
The other open-ended projects geared for young users are channel-backed and filtered via call to `fetchPrivacyProfanityViolations` in `project.js` within `loadProjectBackedLevel_`:
https://github.com/code-dot-org/code-dot-org/blob/ec3ede9c2e23106c0d1d0250c574724d52c1510f/apps/src/code-studio/initApp/project.js#L1747